src/main/java/com/jcraft/jsch/KeyPairECDSA.java

```diff
@@ -351,7 +351,8 @@ public byte[] getSignature(byte[] data) {
               .asSubclass(SignatureECDSA.class);
       SignatureECDSA ecdsa = c.getDeclaredConstructor().newInstance();
       ecdsa.init();
-      ecdsa.setPrvKey(prv_array);
+      // issue 739: prv_array could be destroyed by ecdsa signing process if its first bit is 1
```

Comment:
Can you change this to point at `https://github.com/mwiede/jsch/issues/739` instead of `issue 739`?
That way future readers don't have to figure out how to find what `issue 739` means.

Reply:
done